typescript/cli/src/utils/files.ts

```diff
@@ -15,6 +15,7 @@ import { objMerge } from '@hyperlane-xyz/utils';
 import { log } from '../logger.js';
 
 export const MAX_READ_LINE_OUTPUT = 250;
+export const MAX_ALIAS_YAML = 100_000; // Used for yaml maxAliasCount. Ref: https://eemeli.org/yaml/#tojs-options
```

Comment:
just curiosity but `100_000` has been chosen arbitrarily right?

Reply:
Yup!

Reply:
Some number other than -1 (infinity) is needed to prevent infinite laughs attack